applications/app/views/blindPsCrossword.scala.html

```diff
@@ -0,0 +1,91 @@
+                                @fragments.social(crosswordPage, "top")
+                            </div>
+                            @if(SaveForLaterSwitch.isSwitchedOn) {
+                                <div class="meta__save-for-later js-save-for-later" data-position="top"></div>
+                            }
+                            <div class="meta__numbers modern-visible">
+                                <div class="u-h meta__number" data-discussion-id="@crosswordPage.discussionId" data-commentcount-format="content" data-discussion-closed="@{crosswordPage.isCommentable}">
+                                </div>
+                                <div class="u-h meta__number js-sharecount">
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </header>
+
+            <div class="content__main tonal__main tonal__main--tone-news">
+                <div class="gs-container">
+                    <div class="js-content-main-column">
+                        <div class="crossword__container crossword__container--@crosswordPage.crossword.crosswordType.toString.toLowerCase() crossword__container--blind">
+                            <div class="crossword__blind-row-data-wrapper">
+                                <div class="crossword__blind-data">
+                                    <h3 class="crossword__blind-data--header">Blanks</h3>
+                                        @fragments.blindCrosswordGridData(blankSquares.rows)
+                                </div>
+                            </div>
+                            <div class="crossword__clues">
+                                <div class="crossword__clues--across">
+                                    <h3 class="crossword__clues-header">Across</h3>
+                                    @fragments.blindCrosswordEntries(crosswordPage.crossword.entries.filter(_.direction == "across"))
+                                </div>
+
+                                <div class="crossword__clues--down">
+                                    <h3 class="crossword__clues-header">Down</h3>
+                                    @fragments.blindCrosswordEntries(crosswordPage.crossword.entries.filter(_.direction == "down"))
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </article>
+
+        <div class="content-footer">
+
+            @fragments.discussionFooter(crosswordPage.isCommentable, crosswordPage.isClosedForComments, crosswordPage.shortUrlId)
+
+            @if(crosswordPage.isCommentable) {
+                <div class="js-repositioned-comments content__repositioned-comments"></div>
+            }
+
+            @fragments.onwardPlaceholder()
+        </div>
+    </div>
+}
```

Comment:
more much commonality is there between the standard and this version?  might be possible to extract most of this into a common template